docs/api-reference/next/image.md

```diff
@@ -65,12 +66,13 @@ The `<Image />` component accepts a number of additional properties beyond those
 
 The layout behavior of the image as the viewport changes size.
 
-| `layout`              | Behavior                                                 | `srcSet`                                                                                                    | `sizes` |
-| --------------------- | -------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- | ------- |
-| `intrinsic` (default) | Scale *down* to fit width of container, up to image size | `1x`, `2x` (based on [imageSizes](#image-sizes))                                                            | N/A     |
-| `fixed`               | Sized to `width` and `height` exactly                    | `1x`, `2x` (based on [imageSizes](#image-sizes))                                                            | N/A     |
-| `responsive`          | Scale to fit width of container                          | `640w`, `750w`, ... `2048w`, `3840w` (based on [imageSizes](#image-sizes) and [deviceSizes](#device-sizes)) | `100vw` |
-| `fill`                | Grow in both X and Y axes to fill container              | `640w`, `750w`, ... `2048w`, `3840w` (based on [imageSizes](#image-sizes) and [deviceSizes](#device-sizes)) | `100vw` |
+| `layout`              | Behavior                                                       | `srcSet`                                                                                                    | `sizes` | Has wrapper and sizer |
+| --------------------- | -------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- | ------- | --------------------- |
+| `intrinsic` (default) | Scale *down* to fit width of container, up to image size       | `1x`, `2x` (based on [imageSizes](#image-sizes))                                                            | N/A     | yes                   |
+| `fixed`               | Sized to `width` and `height` exactly                          | `1x`, `2x` (based on [imageSizes](#image-sizes))                                                            | N/A     | yes                   |
+| `responsive`          | Scale to fit width of container                                | `640w`, `750w`, ... `2048w`, `3840w` (based on [imageSizes](#image-sizes) and [deviceSizes](#device-sizes)) | `100vw` | yes                   |
+| `fill`                | Grow in both X and Y axes to fill container                    | `640w`, `750w`, ... `2048w`, `3840w` (based on [imageSizes](#image-sizes) and [deviceSizes](#device-sizes)) | `100vw` | yes                   |
+| `raw`                 | Insert the image element with no automatic layout behavior     | Behaves like `responsive` if the images has the `sizes` prop, and like `fixed` if it does not               | optional| no                    |
```

Comment:
It seems like "Behaves like `responsive` if the images has the `sizes` prop" isn't true right now because it doesnt automatically stretch to fit the viewport like responsive does. This is because the `width` and `height` props go directly to the underlying `<img>`.

It seems like we need a way to maintain aspect ratio from width/height without setting the attributes to get the true responsive behavior.

Reply:
Maybe instead of trying to make direct comparisons to other layout modes, we can just list what the srcSet is directly for both cases (e.g. '1x', '2x' if sizes is set, otherwise '`640w`, `750w`, ... `2048w`, `3840w` (based on [imageSizes](#image-sizes) and [deviceSizes](#device-sizes))

Reply:
I agree with @kara , the whole purpose of `raw` is to not add layout behaviors.

The "responsive" behavior from the browser is to download the right image version according to `height`, `width`, `sizes` and `srcset` attributes. Nothing should come in the way of of this.

I'd even argue if a user passes `srcset` prop when in `raw` layout, the values from the user should be passed directly to the `<img>` without any modification.

I think a default of `1x`/`2x` or `640w, 750w, ...,3840w` depending if `sizes` is set or not is fine, just don't prevent people from generating their own `srcset`. What I consider should always be `640w, ...3840w` might be different for another person and vice versa. The whole point of `raw` is to give back the power to users, defaults are welcome if not forced on you.